web/src/main/webapp/dataViews/tsViewBase.ts

```diff
@@ -272,6 +320,56 @@ export abstract class TSViewBase extends BigTableView {
         this.histogram(this.getSelectedColNames());
     }
 
+    protected privateHistSelected(): void {
+        if (this.getSelectedColCount() !== 1) {
+            this.page.reportError("Must select 1 column for private histogram");
+            return;
+        }
+
+        let title = "Private Histogram";
+	const d = new Dialog(title, "Plot a private histogram with the specified granularity and privacy budget.");
+        const epsilon = d.addTextField("epsilon", "Privacy budget, epsilon", FieldKind.Double, "0.25",
```

Comment:
As discussed, some of this will move from the menu to the server-side PrivateTableTarget.